tools/clang/include/clang/Frontend/CodeGenOptions.h

```diff
@@ -14,12 +14,13 @@
 #ifndef LLVM_CLANG_FRONTEND_CODEGENOPTIONS_H
 #define LLVM_CLANG_FRONTEND_CODEGENOPTIONS_H
 
+#include "dxc/HLSL/HLSLExtensionsCodegenHelper.h" // HLSL change
```

Comment:
This should go after the standard header includes.

Reply:
Done. (interestingly, this change was done automatically by clang-format!)